export function assertUnreachable(_) {
    throw new Error("Unreachable");
}
